### DISCUSSION TITLE
Add Server "is Starting" and "Crashed" to Server Status Embed Discord!

### DISCUSSION THREAD

**Discussion by InfinityofphoenixTimo** [gameserverapp/Platform#885] [Suggestions]
Add two additional server status messages in Form of embed dc Mails to the Other. We have, Offline, Online, Stop. But it would be better with Starting and Crash Messages..

**Comment by mvaessen** [comment]
While checking this, I noticed that the `starting up` is already a thing. The `Crash recovery` state is added and will be available later today
